app/service/item_service.rb

```diff
@@ -0,0 +1,13 @@
+class ItemService
+
+  def self.all_items
+    response = Faraday.get("http://localhost:3000/api/v1/items")
+    JSON.parse(response.body, symbolize_names: true)
+
+  end
+
+  def self.item(item_id)
+    response = Faraday.get("http://localhost:3000/api/v1/items/#{item_id}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
You can combine the ItemService and the MerchantService since they're both using the same service -- rails_engine. I usually stick to one service per API I'm utilizing.